src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useState } from 'react';
+import { Todo } from '../../type/Todo';
+import usersFromServer from '../../api/users';
+import { getUserById } from '../../services/getUserById';
+
+type Props = {
+  todos: Todo[],
+  onAdd: (newTodo: Todo) => void;
+};
+
+export const TodoForm: React.FC<Props> = ({ todos, onAdd }) => {
+  const [title, setTitle] = useState('');
+  const [hasTitleError, setHasTitleError] = useState(false);
+
+  const handleTitleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const textValue = event.target.value.replace(/[^a-zA-ZіІїЇєЄґҐ а-я]+/gu, '');
```

Comment:
```suggestion
    const textValue = event.target.value
      .replace(/[^a-zA-ZіІїЇєЄґҐ а-я]+/gu, '');
```
move to other line should fix such linter issue